src/lib/features/feature-toggle/feature-toggle-controller.ts

```diff
@@ -949,8 +953,8 @@ export default class ProjectFeaturesController extends Controller {
             return;
         }
 
-        await this.startTransaction(async (tx) =>
-            this.transactionalFeatureToggleService(tx).bulkUpdateEnabled(
+        await this.transactionalFeatureToggleService.transactional((service) =>
```

Comment:
this is an example of a transition from out previous transaction handling mechanism to the current one